components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -185,12 +185,28 @@ - (void)applyStyle:(MDCItemBarStyle *)style {
     [self updateTitleTextColor];
     [self updateInk];
     [self updateSubviews];
+    [self updateTitleLines];
     [self updateTitleFont];
     [self updateTransformsAnimated:NO];
     [self setNeedsLayout];
   }
 }
 
+- (void)updateTitleLines {
+  // The presence of an image restricts titles to a single line
+  if (_imageView && !_imageView.hidden) {
+    _titleLabel.numberOfLines = 1;
+  } else {
+    _titleLabel.numberOfLines = _style.textOnlyTitleNumberOfLines;
+  }
+  
+  if (_titleLabel.numberOfLines == 1) {
```

Comment:
Nit: I’m not sure splitting this across more lines adds any clarity in this case. Replace with a brief ternary and comment?

Reply:
Done